### PR TITLE
Added examples

### PR DESCRIPTION
**Pull request by miguelhar**

### CODE REVIEW

**Review by ddl-dclegg — APPROVED** [review]
I think that this looks good to me, we just need to handle the linting check errors

**Comment by steved** [comment]
Any way we can integrate these examples into CI?

**Comment by miguelhar** [comment]
> Any way we can integrate these examples into CI?

Yes, that was the plan, but want to get these in as part of 5.6.0, so we can reference the example in the release notes.

File: examples/public-access/versions.tf [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    }
+    tls = {
+      source  = "hashicorp/tls"
+      version = ">= 3.4.0"
+    }
+  }
+}
+
+provider "aws" {
+
```
[the remark this answers is not in the document]
Comment (ddl-dclegg) [reply examples/public-access/versions.tf:20]: That brings back tags that are already defined in the provider, but I'm talking about promoting the practice of defining [default tags](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#default_tags-configuration-block) in the provider, like
```hcl
provider "aws" {
  default_tags = {
    ...
  }
}
```
Again, it's not a blocker, and maybe I'm just feeling the pain of resources not being cleaned up due to improper tagging, but I figured it made sense to bring it up.

**Pull request merged**